test/k8sT/Services.go

```diff
@@ -537,8 +537,12 @@ var _ = Describe("K8sServicesTest", func() {
 						lbIP, err := kubectl.GetLoadBalancerIP(
 							helpers.DefaultNamespace, "test-lb", 30*time.Second)
```

Comment:
Maybe let's rename `test-lb` to `test-lb-tcp`?

Reply:
Done!